2.0.3's upgrade command doesn't work with `noupgrade`

Trying to compile with `noupgrade` I get:
```
# github.com/twpayne/chezmoi/cmd
cmd/upgradecmd.go:20:13: c.runUpgradeCmd undefined (type *Config has no field or method runUpgradeCmd)
```

--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,7 @@ build-darwin:
 .PHONY: build-linux
 build-linux:
 	GOOS=linux GOARCH=amd64 $(GO) build -o /dev/null .
+	GOOS=linux GOARCH=amd64 $(GO) build -tags=noupgrade -o /dev/null .
 
 .PHONY: build-windows
 build-windows:

--- a/cmd/noupgradecmd.go
+++ b/cmd/noupgradecmd.go
@@ -1,3 +1,5 @@
+// +build noupgrade windows
+
 package cmd
 
 import (

